requirements.txt

```diff
@@ -1,4 +1,5 @@
 branca>=0.6.0
+fiona
```

Comment:
@hansthen this got merged with this added dependency in the requirements. I don’t think that’s right, since it’s not a required dependency. Could you maybe make a PR to revert this?